### PR TITLE
DEP-3489: Added detail to README file

### PR DESCRIPTION
**Pull request by tonymyers**
# DEP-(StoryNumberHere)

## Checklist PR Raiser
 - [ ]  I've ensured code coverage has not decreased
 - [ ]  I've dealt with any new compilation warnings
 - [ ]  I've ensured the team's coding standards have been met (TBC)?
 - [ ]  If relevant, I've created corresponding app-config-XXX changes?
 - [ ]  If I've created new test data, I've ensured there is no real person's data (see https://confluence.tools.tax.service.gov.uk/display/CD/Test+Data+in+the+Open) 
 - [ ]  I've considered the impact my changes have on the UI/Journey tests (please do not break them)
-  [ ]  If relevant, I've added links to associated PRs

## Checklist PR Reviewer
 - [ ]  I've checked to ensure all relevant unit tests have been written
 - [ ]  I've checked to ensure the team's coding standards have been met
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  If I merge I will ensure I use squash and merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,53 @@
...
+and do not require content on the page.
+
+### Version 1
+Known as DAv1, Webchat version 1 or Popup chatskin. Provides a Nuance popup chatskin used for both Webchat and Digital Assistant.
+
+### Version 2
+Known as DAv2, CUI or Embedded chatskin. Provides a full page Embedded chatskin.
+
+### Version 3
+Known as DAv3, CIAPI or HMRC chatskin. This was developed mainley to covercone Accessability issues with the Nuance chatcking
```
[the remark this answers is not in the document]
Comment (tonymyers) [reply README.md]: Removed

File: README.md [hunk 2]
```diff
@@ -1,7 +1,53 @@
 
 # digital-engagement-platform-partials
 
-This is a placeholder README.md for a new repository
+This service provides container content element, required DataPass attributes and Nuance framework script or can 
+pull in a HMRC chatskin.
+
+There are three versions of the container elements that can be used. Versions one and two pull in Nuance chat skins,
```
[the remark this answers is not in the document]
Comment (tonymyers) [reply README.md:7]: added method

File: README.md [hunk 3]
```diff
@@ -1,7 +1,53 @@
...
+
+### Reactive chatskin
+Is where a user is expected to complete an action to trigger a webchat or Digital Assistand. For example click on 
+a button or a link.
+
+### Proactive chatskin
+Is where no action is required for a user to take action to trigger a webchat or Virtual Assistant 
+
+### Popup chatskin
+A chatskin that is generally used in other MDTP services. A Popup skin sits in the bottom right of a page and does not
```
[the remark this answers is not in the document]
Comment (tonymyers) [reply README.md]: Removed

File: README.md [hunk 4]
```diff
@@ -1,7 +1,53 @@
...
+### Popup chatskin
+A chatskin that is generally used in other MDTP services. A Popup skin sits in the bottom right of a page and does not 
+obscure any content on the page the service uses.
+
+### Embedded chatskin
+A chatskin that takes up the whole paged and is only used in digital-engagement-platform-frontend. This service serves links from gov.uk 
+and do not require content on the page.
+
+### Version 1
+Known as DAv1, Webchat version 1 or Popup chatskin. Provides a Nuance popup chatskin used for both Webchat and Digital Assistant.
```
[the remark this answers is not in the document]
Comment (tonymyers) [reply README.md]: Simplified

**Review by pratikrana06 — APPROVED** [review]
Approved

**Pull request merged**